android/src/com/mapswithme/maps/Framework.java

```diff
@@ -61,6 +61,13 @@
     public boolean buildings;
   }
 
+  public static class RouteAltitudeLimits
+  {
+    public int minRouteAltitude;
```

Comment:
Guys, let's do it in the master after creating new release branch. Volodya, please setup a task for this.